app/views/groups/index.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<p style="color: green"><%= notice %></p>
+<%= link_to 'Home', user_path(@user.id), class: "btn btn-primary" %>
+
```

Comment:
- [ ] After clicking on logout the page is not changing though the user is `logout` but they are not redirecting to `sign up` or `sign in` page after `logout`. Please fix this.

Reply:
thanks for the review. change has been implemented